qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -134,19 +132,64 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
             }
             hasNetworkViolation = results.any { isNetworkViolation(it) }
             hasProcessViolation = results.any { isProcessViolation(it) }
+            // TODO: Remove debug log
+            log.info "Found violations in Splunk: \n${results}"
+            log.info "hasNetworkViolation: ${hasNetworkViolation}\nhasProcessViolation: ${hasProcessViolation}"
+            // TODO: /Remove debug log
             if (hasNetworkViolation && hasProcessViolation) {
                 log.info "Success!"
                 break
             }
         }
 
+        // After we know that violations arrived, manually kick off search that converts them into alerts
+        postToSplunk(port, "/services/saved/searches/" +
+                "Threat%20-%20Create%20Notable%20from%20RHACS%20Alert%20-%20Rule/dispatch", [
+                "dispatch.now": "true",
+                "force_dispatch": "true",
+        ])
+
+        // Check for Alerts
+        List<Map<String, String>> alerts = Collections.emptyList()
+        boolean hasNetworkAlert = false
+        boolean hasProcessAlert = false
+        for (int i = 0; i < 20; i++) {
+            log.info "Attempt ${i} to get Alerts from Splunk"
+            // Hint: If this produces no results, evaluate expanding the earliest_time, e.g. to -15m
+            def vSearchId = SplunkUtil.createSearch(port, "| from datamodel Alerts.Alerts")
+            TimeUnit.SECONDS.sleep(15)
+            Response vResponse = SplunkUtil.getSearchResults(port, vSearchId)
+            // We should have at least one violation in the response
+            if (vResponse == null) {
+                continue
+            }
+            alerts = vResponse.getBody().jsonPath().getList("results")
+            if (alerts.isEmpty()) {
+                continue
+            }
+            hasNetworkAlert = alerts.any { isNetworkViolation(it) }
+            hasProcessAlert = alerts.any { isProcessViolation(it) }
+            // TODO: Remove debug log
+            log.info "Found Alerts in Splunk: \n${alerts}"
+            log.info "hasNetworkAlert: ${hasNetworkAlert}\nhasProcessAlert: ${hasProcessAlert}"
+            // TODO: /Remove debug log
+            if (hasNetworkAlert && hasProcessAlert) {
+                log.info "Success!"
```

Comment:
Could we write a bit more what kind of success is this? Reading the code gives some hints, but let's make sure that reading the logs give the same - for example: "Success: Found network and process alert".